File: ePozoriste.Model/Pozoriste.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ePozoriste.Model
{
    public class Pozoriste
    {
        public int PozoristeId { get; set; }
        public string Naziv { get; set; }
        public string Adresa { get; set; }
        public string Webstranica { get; set; }
        public string Email { get; set; }
        public string BrTelefona { get; set; }
        public string? Logo { get; set; }
        public bool Aktivan { get; set; }

        public int? GradId { get; set; }
        public virtual Grad? Grad { get; set; }
    }
}

File: epozoriste_mobile/lib/models/pozoriste.dart
import 'package:epozoriste_mobile/models/models.dart';
import 'package:json_annotation/json_annotation.dart';

part 'pozoriste.g.dart';

@JsonSerializable()
class Pozoriste {
  int pozoristeId;
  String naziv;
  String adresa;
  String webstranica;
  String email;
  String brTelefona;
  String? logo;
  bool aktivan;
  int? gradId;
  Grad? grad;

  Pozoriste(
      {required this.pozoristeId,
      required this.naziv,
      required this.adresa,
      required this.webstranica,
      required this.email,
      required this.brTelefona,
      this.logo,
      required this.aktivan,
      this.gradId,
      this.grad});

  factory Pozoriste.fromJson(Map<String, dynamic> json) =>
      _$PozoristeFromJson(json);
  Map<String, dynamic> toJson() => _$PozoristeToJson(this);
}
